ledger/src/shred/merkle.rs

```diff
@@ -1289,7 +1290,7 @@ fn finish_erasure_batch(
             &Shred::from_payload(shred).unwrap()
         });
     }
-    Ok(root)
+    Ok(tree)
```

Comment:
What is the benefit of returning the entire tree here?

Reply:
Just an intuition.  I figured we have the tree computed here and returning just the root (and dumping the tree) or returning the whole tree has the same cost.  So somehow this change felt more "future proof".

Reply:
Reverted.